R/rwebppl.R

```diff
@@ -343,21 +344,20 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
   if (file.exists(error_file)) {
     file.remove(error_file)
   }
+  finish_file <- sprintf("/tmp/webppl_finished_%s", uid)
+  finish_arg <- finish_file
+  if (file.exists(finish_file)) {
+    file.remove(finish_file)
+  }
 
   # run rwebppl JS script with model file and packages as arguments
   # any output to stdout gets sent to the R console while command runs
-  system2(script_path, args = c(file_arg, output_arg, package_args),
-          stdout = "", stderr = error_file, wait = FALSE)
+  system2(script_path, args = c(file_arg, output_arg, finish_arg, package_args),
```

Comment:
It took me a while to figure out what was going on with these args. Could we switch to using [Optimist](https://github.com/substack/node-optimist) to do argument parsing? Then we could pass each of these args with a flag making the difference between "finish_arg" and "finish_file" clearer.